### ISSUE TITLE
Is there a rotation listener?

### ISSUE THREAD

**Issue by GolgothSchumi** [dmitry-zaitsev/CircleLayout#9]
Very nice library. I would like to ask if this layout has a rotation listener. For example, I have 4 view and I swipe my fingers on the screen, the layout should perform a form of rotation animation.

**Comment by dmitry-zaitsev** [comment]
No, there is no such thing as for now. Layout in current implementation is
not really suitable for layout animations - that's something that can be
improved.
